### DISCUSSION TITLE
My Github Account got suspended suddenly

### DISCUSSION THREAD

**Discussion by Priyanshux085** [community/community#134351] [Other Feature Feedback, Questions, & Ideas]
Hello GitHub Support Team,

My name is Priyanshu, and my GitHub username is Priyanshu085. I am writing to request an appeal and reinstatement of my GitHub account, which has been banned due to a violation.

Recently, I was completing a form related to the GitHub Student Developer Pack and got banned suddenly, without any warning or clear explanation of the specific violation. I rely heavily on my GitHub account for hosting and managing my academic and personal projects. Being unable to access my account is significantly impacting my development work and learning.

I understand the importance of adhering to GitHub’s Terms of Service and Community Guidelines. If there were any inadvertent violations on my part, I sincerely apologize and am committed to ensuring full compliance in the future.

Please review my case and consider reinstating my account. If further information is needed, I am more than willing to provide it.

Thank you for your understanding and assistance.

Best regards,  
Priyanshu  
GitHub Username: Priyanshu085  
Email: abpriyanshu007@gmail.com

**Comment by SAHELISAHAA** [comment]
@Priyanshux085 same thing happened with me

**Comment by Davdadev** [comment]
try https://support.github.com/contact/account

**Reply by Priyanshux085** [reply]
Thanks @Davdadev

**Comment by ygetelcare** [comment]
Same here, apparently just connecting CoPilot in VSCode will trigger some bug in their violation screening and get accounts suspended, it happened to a new account I created, yge2020, just for trying CoPilot.  How ridiculous, this is technically fraudulent business practice, we need to report them to the BBB.

**Comment by JoshuaDevCo** [comment]
Same here, and they don't even reply to the ticket, lol… I'm waiting almost 3 months.
